src/embeddedExplorer/setupEmbedRelay.ts

```diff
@@ -94,6 +97,33 @@ export function setupEmbedRelay({
           });
         }
       }
+
+      // If the user is executing a subscription...
+      if (isSubscription && data.operation && data.operationId) {
+        // Extract the operation details from the event.data object
+        const {
+          operation,
+          operationId,
+          operationName,
+          variables,
+          headers,
+          subscriptionUrl,
```

Comment:
For the endpoint url (non subscription) we have that in the embed config (passed in via `embeddable-explorer` / this repo). 

That is the case because in the embeddable explorer, we don't necessarily always have `graphRef`. Folks can pass in their schema manually, so there isn't always a subscriptionUrl that we can pull here. 

I think we want to follow the same pattern for subscriptions. We would need to add `subscriptionUrl` and `handleSubscription`(return an Observable) to the `embeddable-explorer` repo & use those in the fetching on the `embeddable-explorer` side.

This might be a big ish change - feel free to merge it into your existing commits. It would make it easier for me to review!

Reply:
Makes sense! I'll add that in a new PR

Reply:
@William010x mind if we add it into this PR 🙏 I was going to wait until this was fixed up to give a full test

Reply:
Also just noting this will change the studio-ui pr a touch

Reply:
Not sure if what I added for `handleSubscription` is exactly what you had in mind. It takes the `{ query, variables, operationName }` and `client` as parameters, then returns the Observable, which subscribe gets called on. Also, should I be using the `Observable` from @apollo/client instead?